python/tvm/relay/tensorrt.py

```diff
@@ -220,24 +220,30 @@ def register_tensorrt_annotations(trt_version, use_implicit_batch=True):
     # _register_external_op_helper("slice_like")
 
     def add_whitelist_fn(attrs, args):  # pylint: disable=unused-variable
+        static_add = True
         for arg in args:
-            if not arg.checked_type.shape:
+            if not arg.checked_type.shape[1:]:
```

Comment:
Will this cause failure if shape is empty - scalar?

Reply:
Hmm, let me add a test case for this, I was thinking that if its a scalar shape would be an empty list rather than None

Reply:
What about tensors of dimension 1 ?

Reply:
If its 1-d, then its just the batch dimension. So when it goes to TRT with implicit batching. it would become a scalar. Not sure if we have handled that situation in TRT - but since we have already been letting them through we can do `len(arg.checked_type.shape) < 1` for now

Reply:
Done